jobs/acceptance-tests-brain/templates/pre-start.erb

```diff
@@ -5,6 +5,9 @@ set -o nounset
 
 PATH=$PATH:/var/vcap/packages/cli/bin
 
-cf api --skip-ssl-validation "https://api.$DOMAIN"
-cf auth admin "${CLUSTER_ADMIN_PASSWORD}"
+cf api --skip-ssl-validation \
+   "https://<%= properties.smoke_tests.api.shellescape %>"
```

Comment:
I think we need to `<% require 'shellwords' %>` at the top of the file? (Of course, if this actually works then that's fine.)

Also, do we use [`acceptance_tests_brain.domain`](https://github.com/SUSE/brain-tests-release/blob/fe15bb7fa6e9f681644dbb02517296ae9af0121e/jobs/acceptance-tests-brain/spec#L23) anywhere?

Also, please unquote things that have already been `shellescape`d; that would result in double-quoting.  Sometimes.

Reply:
The `acceptance_tests_brain.domain` is translated into `CF_DOMAIN`, which in turn is used in six tests, as well as the `testutils.rb` used by all the tests.

Will make changes. Will see if I can test this locally before making the next commit.

Reply:
Addressed with commit c8efc353b20a4b368a2a0a509ce4fbf92eff7b03.